extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -375,7 +375,8 @@ function getAnchorElements(rootElement, anchorObj) {
     user().warn(TAG, 'No selector in anchor');
     return [];
   }
-  let elements = [].slice.call(scopedQuerySelectorAll(rootElement, selector));
+  let elements = [].slice.call(scopedQuerySelectorAll(
```

Comment:
I think we should just switch to `rootElement.querySelectorAll()`. There's really no value for scoped selectors for document-wide search. But the code will be less ambiguous.